pkg/config/config.go

```diff
@@ -17,13 +17,21 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
-// ChaosServerConfig defines the configuration for Chaos Dashboard
-type ChaosServerConfig struct {
+// ChaosDashboardConfig defines the configuration for Chaos Dashboard
+type ChaosDashboardConfig struct {
 	ListenHost           string `envconfig:"LISTEN_HOST" default:"0.0.0.0"`
 	ListenPort           int    `envconfig:"LISTEN_PORT" default:"2333"`
 	MetricAddress        string `envconfig:"METRIC_ADDRESS"`
 	EnableLeaderElection bool   `envconfig:"ENABLE_LEADER_ELECTION"`
 	Database             *DatabaseConfig
+	PersistTTL           PersistTTLConfig
+}
+
+// PersistTTLConfig defines the configuration of ttl
+type PersistTTLConfig struct {
+	SyncPeriod string `envconfig:"TTL_SYNC_PERIOD" default:"12h"`
+	Event      string `envconfig:"TTL_EVENT"       default:"168h"`
+	Experiment string `envconfig:"TTL_EXPERIMENT"  default:"336h"`
```

Comment:
The comment of the ParseDuration function says: "Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".   "  .
So I used "h" instead of "d".

Reply:
Yes, you are right. Sorry for my review. But to my surprise the go standard library doesn't support `d`